pipeline/01_parser.py

```
@@ -1,89 +1,533 @@
+        rows_md: list[str] = []
+        tr_tag  = "{http://www.hancom.co.kr/hwpml/2012/paragraph}tr"
+        tc_tag  = "{http://www.hancom.co.kr/hwpml/2012/paragraph}tc"
+        for tr in tbl_elem.iter(tr_tag):
+            cells = [_elem_text(tc).replace("\n", " ").strip() for tc in tr.iter(tc_tag)]
+            if any(cells):
+                rows_md.append("| " + " | ".join(cells) + " |")
+        if not rows_md:
+            return ""
+        header    = rows_md[0]
+        separator = "| " + " | ".join(["---"] * (header.count("|") - 1)) + " |"
+        return "\n".join([header, separator] + rows_md[1:])
```

Ruff (0.15.14)

[warning] Consider `[header, separator, *rows_md[1:]]` instead of concatenation

Replace with `[header, separator, *rows_md[1:]]`

(RUF005)